MetaMask/metamask-mobile
# chore(Sentry): filter out Route Change

## **Description**

This PR removes Route Change from being logged as a performance metric. Route Change is an event that does not provide any value when it comes to performance and is inflating the cost of performance monitoring. 

## **Related issues**

Fixes: https://github.com/MetaMask/mobile-planning/issues/1394

## **Manual testing steps**

1. Run application
2. See performance events logged and Route Change not being called

## **Screenshots/Recordings**

NA

### **Before**

NA

### **After**

[Log output video](https://recordit.co/BELvtdc4qF)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
